cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/goals/PreferredLeaderElectionGoal.java

```diff
@@ -111,6 +111,8 @@ public boolean optimize(ClusterModel clusterModel, Set<Goal> optimizedGoals, Opt
         }
       }
     }
+    // This goal is optimized in one pass.
```

Comment:
Not introduced by this patch: at line 98, number of `{}` is one more than arguments.